pyramid_swagger/tween.py

```diff
@@ -542,10 +542,10 @@ def validate_response(response, validator_map):
 
 
 def prepare_body(response):
-    # content_type and charset must both be set to access response.text
-    if response.content_type is None or response.charset is None:
+    # content_type must be set to access response.text
+    if not response.content_type:
```

Comment:
I'm a little bit confused as your travis build says it's testing Pyramid 1.7 and Webob 1.6 and fails, so I'm assuming you're not reproducing the issue you're describing...?

But anyway, if that was indeed using older versions of those libraries, that is a general incompatibility, right? Nothing that changed with this PR, as I'm not setting or unsetting charsets. And your test only fails as you mock the response object. It will never fail in "reality" as we use the Response object from `Webob`, so it will always be configured the proper way (either with or without content type).